src/pages/MeetingPage/MeetingPage.tsx

```diff
@@ -74,8 +76,11 @@ const MeetingPage = () => {
     const email = formEntries.find(entry => entry[0] === 'email')?.[1]
     const phone = formEntries.find(entry => entry[0] === 'phone')?.[1]
     const referal = formEntries.find(entry => entry[0] === 'referal')?.[1]
-    const fields = formEntries.filter(entry => entry[0] === 'field').flatMap(entry => entry[1].toString().split('/'))
+    const fields = filterRepeatValues(
+      formEntries.filter(entry => entry[0] === 'field').flatMap(entry => entry[1].toString().split('/')),
+    )
```

Comment:
I think we can reduce our dependence on the helper ,when this method can be done easily and directly

`const fields = [...new Set(formEntries.filter(entry => entry[0] === 'field').flatMap(entry => entry[1].toString().split('/')))]; `

or , If it's for readability , we can use `uniqueFields` instead of `fields` for variable name

Reply:
Okay, I'll make the changes as you said first.
However, I think we can discuss why we need to rely less on assistants.
Originally, I was considering reuse and readability, so I chose to make a function and export it from the helper.

Reply:
https://www.npmjs.com/package/ramda
https://www.npmjs.com/package/rambda -> win
https://www.npmjs.com/package/lodash

use utility library more often instead of making wheels again